python/servo/build_commands.py

```diff
@@ -123,7 +123,9 @@ def build(self, build_type: BuildType, jobs=None, params=None, no_package=False,
             for key in env:
                 print((key, env[key]))
 
-        self.download_and_build_android_dependencies_if_needed(env)
+        # FIXME: disabled until GStreamer support is enabled
+        # self.download_and_build_android_dependencies_if_needed(env)
+
```

Comment:
Let's just remove this for now as well as `download_and_build_android_dependencies_if_needed()`.